src/components/Subthoughts.tsx

```diff
@@ -462,8 +462,10 @@ export const SubthoughtsComponent = ({
     1. Force actualDistance to 2 to hide thoughts.
     2. Set zoomCursor and zoomParent CSS classes to handle siblings.
   */
-  const zoomCursor = cursor && attribute(state, pathToContext(cursor), '=focus') === 'Zoom'
-  const zoomParent = cursor && attribute(state, pathToContext(contextOf(cursor)), '=focus') === 'Zoom'
+  const zoomCursor = cursor && (attribute(state, pathToContext(cursor), '=focus') === 'Zoom'
+    || attribute(state, pathToContext(contextOf(cursor)).concat('=children'), '=focus') === 'Zoom')
+  const zoomParent = cursor && (attribute(state, pathToContext(contextOf(cursor)), '=focus') === 'Zoom'
+    || attribute(state, pathToContext(contextOf(contextOf(cursor))).concat('=children'), '=focus') === 'Zoom')
```

Comment:
This works, but if you do it this way, then each child will have to individually check its parent. It would make more sense if the parent checked itself and then passed the result to its children. This is the way that `=children/=style` currently works.